src/wp-includes/class-wp-oembed.php

```diff
@@ -414,6 +414,11 @@ public function get_html( $url, $args = '' ) {
 			return false;
 		}
 
+		// Lazy load oEmbed iframes.
+		if ( wp_lazy_loading_enabled( 'iframe', 'oembed' ) && isset( $data->html ) && str_starts_with( $data->html, '<iframe' ) && ! str_contains( $data->html, 'loading' ) ) {
+			$data->html = wp_iframe_tag_add_loading_attr( $data->html, 'oembed' );
+		}
```

Comment:
Is this the right place for this code? I believe the output here will get cached in `WP_Embed` which means (1) it won't correctly respect the context in `wp_iframe_tag_add_loading_attr` and (2) it won't apply to previously-cached embeds. 

It could instead be added as an `embed_oembed_html` filter which operates on the cached HTML.

Reply:
Hmmm, good point, I'll try moving it there.

Reply:
Moved. Needs more testing to make sure this works as expected.